content/docs/rfcs/32/README.md

```diff
@@ -0,0 +1,565 @@
+---
+slug: 32
+title: 32/RLN
+name: RLN specification
+status: raw
+editor: Blagoj Dimovski <blagoj.dimovski@yandex.com>
+contributors: <TO BE ADDED>
+---
+
+# Abstract
+
+The following document covers the specification of the RLN construct as well as the auxiliary libraries necessary for interaction with it.
+
+# Motivation
+
+Spamming is the use of messaging systems to send multiple unsolicited messages (spam) to large numbers of recipients for the purpose of commercial advertising, for the purpose of non-commercial proselytizing, for any prohibited purpose (especially the fraudulent purpose of phishing), or simply sending the same message over and over to the same user. 
+Spamming by certain entities can worsen the user experience for the other entities in the application, 
+and in some cases can be detrimental for the applications.
+
+RLN (Rate limiting nullfier) is a construct based on zero-knowledge proofs that enables spam prevention mechanism for decentralized, anonymous environments. 
+In anonymous environments, the identity of the entities is unknown, and thus it is very hard to prevent spam attacks. 
+Many applications would benefit from anonymity, but the potential for spam could worsen their UX to a large extent. 
+RLN allows for anonymous signaling while enabling spam prevention on protocol level.
+
+
+# Technical overview
+
+The main RLN construct is implemented using a [ZK-SNARK](https://z.cash/technology/zksnarks/) circuit. 
+However it is helpful to describe the other necessary outside components for interaction with the circuit, 
+which together with the ZK-SNARK circuit enable the above mentioned features.
+
+
+## Terminology
+
+| Term                      | Description                                                                         |
+|---------------------------|-------------------------------------------------------------------------------------|
+| **ZK-SNARK**    | https://z.cash/technology/zksnarks/              |
+| **Stake**                 | Financial or social stake required for registering in the RLN applications. The staking requirement is necessary for disincentivising the end users to spam. A common stake examples are: locking cryptocurrency (financial), linking reputable social identity. |
```

Comment:
I think this part can be removed safely to keep the table more succinct, it is already explained in the text 
```suggestion
| **Stake**                 | Financial or social stake required for registering in the RLN applications.  Common stake examples are: locking cryptocurrency (financial), linking reputable social identity. |
```